.github/workflows/CI2.yml

```diff
@@ -235,21 +258,25 @@ jobs:
         if: ${{ matrix.heffte != 'NoheFFTe' }}
         # actions/checkout doesn't work for external repos yet
         run: |
-          git clone --depth 1 --branch v2.0.0 https://bitbucket.org/icl/heffte.git heffte
+          git clone --depth 1 --branch v2.1.0 https://bitbucket.org/icl/heffte.git heffte
       - name: Build heffte
         if: ${{ matrix.heffte != 'NoheFFTe' }}
         working-directory: heffte
         run: |
-          [[ ${{ matrix.heffte }} == "heFFTe_FFTW" ]] && heffte_cmake_opts+=( -DHeffte_ENABLE_FFTW=ON )
-          [[ ${{ matrix.heffte }} == "heFFTe_MKL" ]] && heffte_cmake_opts+=( -DHeffte_ENABLE_MKL=ON )
+          [[ ${{ matrix.heffte }} == *"heFFTe_FFTW"* ]] && heffte_cmake_opts+=" -DHeffte_ENABLE_FFTW=ON"
+          [[ ${{ matrix.heffte }} == *"heFFTe_MKL"* ]] && heffte_cmake_opts+=" -DHeffte_ENABLE_MKL=ON"
+          if [[ ${{ matrix.heffte }} == *"ONEAPI"* ]]; then
+            heffte_cmake_opts+=" -DHeffte_ENABLE_ONEAPI=ON"
+            source /opt/intel/oneapi/setvars.sh
```

Comment:
Instead of doing that we should really set the right env variables in the container.....

Reply:
> Instead of doing that we should really set the right env variables in the container.....

This script is part of Intel OneAPI and that's what they want people to use. You can source that inside the container build, but tracking each and every variable that Intel sets will be too much in my opinion.

Reply:
The problem with sourcing it here is that it won't persist if we ever deploy the container. Plus if we fix it in the kokkos base container other consumers will profit from it as well.

Here are the lines where we setup the intel environment: 
https://github.com/kokkos/ci-containers/blob/main/fedora#L28-L29

Reply:
Reading https://github.com/ECP-copa/Cabana/pull/448#discussion_r699419351, the solution might be as simple as adding the location of `libOpenCL.so` to `LD_LIBRARY_PATH`.

Reply:
I was trying to get the SYCL build working before cleaning this up, but we're failing well before the FFT tests. We may need to separate that into a new PR (which also pushes off this sourcing issue)